Mergin/version_viewer_dialog.py

```diff
@@ -0,0 +1,588 @@
+from collections import deque
```

Comment:
Missing licence header

Reply:
I added one, but there is a lot of missing header in the plugin, I'll do a separate PR to correct it